recipes/base_install.rb

```diff
@@ -22,10 +22,15 @@
   include_recipe 'yum'
   if node['platform_family'] == 'amazon'
     alinux_extras_topic 'epel'
-  elsif node['platform_version'].to_i < 7
+  elsif node['platform_family'].to_i == 'rhel'
```

Comment:
Can we double check that this condition is correct? Looks like we are trying to convert `node['platform_family']` to integer?

Reply:
yup you're right, fixed.